.gitignore

```diff
@@ -3,4 +3,7 @@
 
 # ignore files
 c4t-java.iml
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# ignore prettier files
```

Comment:
Could you actually revert the gitignore back to what it was before? We need to have the prettier file since it configures the github actions bot.

Reply:
fixed, I thought this was auto-generated by my IDE